content/en/upcoming-features.md

```diff
@@ -4,12 +4,6 @@ slug: upcoming-features
 top_graphic: 1
 ---
```

Comment:
please add `lastmod` to detect old translations

Reply:
lastmod: 2019-06-15
like this?

Reply:
perfect!